example/src/screens/APIClientScreen.tsx

```diff
@@ -147,17 +148,47 @@ const RequestHeader = ({ index, header, updateHeader }: RequestHeaderProps) => {
 export default function APIClientScreen({ route }: APIClientScreenProps) {
     const { client } = route.params;
 
-    const [method, setMethod] = useState("GET");
-    const [endpoint, setEndpoint] = useState("/");
-    const [timeoutInterval, setTimeoutInterval] = useState("");
-    const [body, setBody] = useState("");
+    const [method, setMethod] = useState(METHOD.POST);
+    const [endpoint, setEndpoint] = useState("/api/v4/users/login");
+    const [timeoutInterval, setTimeoutInterval] = useState(30);
+    const [body, setBody] = useState('{"login_id":"","password":""}');
```

Comment:
I'm wondering if it's worth also setting a "default" `APIClient` that has MatterMost API details prefilled, vs on the creator here? So when you open the example app - you see a generic client, a mattermost api client and can add a client with blanks.